integration/test_auth.py

```diff
@@ -372,6 +373,37 @@ def test_import_users_with_password(api_key):
     finally:
         auth.delete_user(uid)
 
+@pytest.fixture
```

Comment:
This object can be initialized in the test cases where it's needed. No need to turn it into a fixture.

Reply:
Ok. may be my understanding is wrong - I thought its instantiated only when its injected in a method.

Reply:
It is, but this object is cheap to instantiate. Fixtures are best for heavy objects.